integration-tests/README.md

```diff
@@ -0,0 +1,295 @@
+# Integration Tests for Operator
+
+This documentation describes the functional use cases that are covered in integration testing for Weblogic Operator. The tests are written in java(Junit tests) and driven by maven profile. All the use cases covered in run.sh will be covered in java integration tests. Currently, QUICKTEST use cases and some of non-quick test(till domain4) use cases are covered in java tests.
```

Comment:
Full name is "the Oracle WebLogic Server Kubernetes Operator".   

"java" -> "Java"
"Junit" -> "JUnit"
"maven" -> "Maven"

Need a space after java in second line.

Reply:
Done